core/client/helpers.js

```diff
@@ -207,6 +208,8 @@ const addToSession = (key, values) => {
 };
 
 registerModules = modules => addToSession('modules', modules);
+registerMainModules = modules => addToSession('mainModules', modules);
+registerGameModules = modules => addToSession('gameModules', modules);
```

Comment:
It's getting complicated I think, I don't see the difference between "modules" and "mainModules" 🤔. If it was just to have a different interface wouldn't it be better to just add a class on the parent selector?

Reply:
Component are not inserted at the same place into the DOM